build.zig

```diff
@@ -1,18 +1,32 @@
 const std = @import("std");
 
 pub fn build(b: *std.Build) void {
+    // get optimize and target
     const optimize = b.standardOptimizeOption(.{});
     const target = b.standardTargetOptions(.{});
 
-    _ = b.addModule("ini", .{
-        .root_source_file = .{
-            .path = "src/ini.zig",
-        },
+    // expose the ini module
+    const ini = b.addModule("ini", .{
+        .root_source_file = b.path("src/ini.zig"),
     });
 
+    // create c lib
+    const c_lib = createCLib(b, optimize, target);
+
+    // for zig example
+    example(b, optimize, target, ini);
+
+    // for c example
+    cExample(b, optimize, target, c_lib);
+
+    // for uint test
+    unitTest(b, optimize, target, ini, c_lib);
```

Comment:
Please merge these functions into the `build` fn, and use the following order of decls:

- Steps
- Build Options
- Dependencies (if any)
- Build Logic